javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,197 @@
+const firstName = document.getElementById("firstName");
+const surName = document.getElementById("surName");
+const fullNameButton = document.getElementById("fullNameButton");
+const showFullName = document.getElementById("showFullName");
+
+function fullName() {
+  showFullName.innerHTML = `${firstName.value} ${surName.value}`;
+  showFullName.classList.add("show");
+  return showFullName;
+}
+
+/*
+
+function fullName(firstName,surName){
+    return `${firstName} ${surName}`;
+}
+ */
+
+// Formal FullName
+
+const firstFormalName = document.getElementById("firstFormalName");
+const surFormalName = document.getElementById("surFormalName");
+const fullFormalNameButton = document.getElementById("fullFormalNameButton");
+const showFullFormalName = document.getElementById("showFullFormalName");
+const useFormalName = document.getElementById("useFormalName");
+
+function fullFormalName(firstFormalName, surFormalName, useFormalName) {
```

Comment:
Its really nice to see that you have also added HTML & CSS to the tasks, it shows me that you have Great knowledge of DOM elements and so.
I would actually review the functions and give feedback to it.
Good job 👍 
My feedback would be to focus on variable naming convention as the parameters, here  names should be firstname & surname and based on the boolean for useFormalName, it should be print the name.

Reply:
I can't use firstName and surName as variables, I use them before in another problem in the same javascript file